<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="accessibleDescription">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>241</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="RadioButtons">
     <item>
      <widget class="QRadioButton" name="ButtonCD">
       <property name="text">
        <string>ЦДA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ButtonBF">
       <property name="text">
        <string>Брезенхем(float)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ButtonBI">
       <property name="text">
        <string>Брезенхем(int)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ButtonBS">
       <property name="text">
        <string>Брезенхем с устранением ступенчатости</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ButtonBA">
       <property name="text">
        <string>Библиотечный алгоритм</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ButtonWU">
       <property name="text">
        <string>Алгоритм Ву</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_line">
       <property name="text">
        <string>Цвет линии(текущий цвет-черный)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonBlack">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonWhite">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonYellow">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,255,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonRed">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOrange">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,104,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonPurple">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(128,0,128);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonBlue">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0,0,255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonGreen">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0,255,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Цвет фона</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonBlack_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonWhite_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonYellow_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,255,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonRed_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOrange_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(255,104,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonPurple_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(128,0,128);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonBlue_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0,0,255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonGreen_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0,255,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="Xstart">
       <property name="placeholderText">
        <string>Координата X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Конец отрезка</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Xend">
       <property name="placeholderText">
        <string>Координата X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Начало отрезка</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="Ystart">
       <property name="placeholderText">
        <string>Координата Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Yend">
       <property name="placeholderText">
        <string>Координата Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(400;0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(400;400)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(0;400)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(0;0)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить рисунок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить отрезок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnalyzeStepButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>219</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Анализ ступенчатости</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnalyzeEffButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>219</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Анализ эффективности</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>            Угол поворота(в градусах)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnalyzeButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>219</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить пучок</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Angleline">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>219</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Длина отрезка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>200</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Координата X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Координата Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Длина отрезка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Координаты центра</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Построение пучка:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
